fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1758,7 +1762,16 @@ void ReadYourWritesTransaction::addWriteConflictRange( KeyRangeRef const& keys )
 	writes.addConflictRange(r);
 }
 
-Future<Void> ReadYourWritesTransaction::commit() { 
+Future<Void> ReadYourWritesTransaction::commit() {
+	if (!newClusterFile.empty()) {
+		if (!writes.empty()) {
+			fprintf(stderr,
+			        "Error: If \xff\xff/cluster_file_path is set, it must be the only mutation in its transaction\n");
+		}
+		ASSERT(writes.empty());
```

Comment:
Is it possible for a client to violate this? If so, we should throw an appropriate error in that case (and probably remove the printf).

Reply:
Done.